python/paddle/v2/framework/tests/gradient_checker.py

```diff
@@ -76,6 +95,131 @@ def product(dim):
     return gradient_flat.reshape(tensor_to_check.get_dims())
 
 
+class GradientChecker(unittest.TestCase):
+    def check_grad(self,
+                   forward_op,
+                   input_vars,
+                   inputs_to_check,
+                   output_name,
+                   no_grad_set=set(),
+                   only_cpu=False,
+                   rtol=0.005,
+                   atol=0.05):
```

Comment:
only one augment `rtol` or `atol` is enough. Maybe `rtol` or `delta/epsilon` is a better name to understand.
Because Gradient Check is a numeric way to get the gradient, it is normal to have some diff.